README.rst

```diff
@@ -194,4 +195,9 @@ Contributors:
 .. _Filter: http://docs.python.org/howto/logging-cookbook.html#using-filters-to-impart-contextual-information
 
 .. |Build_Status| image:: https://travis-ci.org/severb/graypy.svg?branch=master
-    :target: https://travis-ci.org/severb/graypy
\ No newline at end of file
+    :target: https://travis-ci.org/severb/graypy
+
+
+.. |Coverage_Status| image:: https://codecov.io/gh/severb/graypy/branch/master/graph/badge.svg
```

Comment:
@severb you may have to do some of your own config with codecov to get this badge to work nicely.